service_discovery_manager/Tools/weightFinder/browser/loadingtime_extract.js

```diff
@@ -0,0 +1,61 @@
+//To get loading time using node.js
+//After opening chrome browser with option --remote-debugging-port=9222
+//usage: node ./filename.js http://www.url.com
+//
+const CDP = require('chrome-remote-interface');
+const fs = require('fs');
+const util = require('util');
+const path = require('path');
+
+const g_duration = 20*1000
```

Comment:
Need a space among operand and operator.

Reply:
Done